### ISSUE TITLE
jslint error for getCurrent User:

### ISSUE THREAD

**Issue by nolandubeau** [strongloop/loopback-sdk-angular-cli#18]
Hi,

I encountered the following jslint error:

```
Expected '{' and instead saw 'id'.
              if (id == null) id = '__anonymous__';


"getCurrent": {
           url: urlBase + "/users" + "/:id",
           method: "GET",
           params: {
             id: function() {
              var id = LoopBackAuth.currentUserId;
              if (id == null) id = '__anonymous__';
              return id;
            },
          },
          interceptor: {
            response: function(response) {
              LoopBackAuth.currentUserData = response.data;
              return response.resource;
            }
          },
          __isGetCurrentUser__ : true
        }
```

I believe this can be corrected by:

```js
if (id == null) {
    id = '__anonymous__';
}
```

thanks.

**Comment by nolandubeau** [comment]
Additionally,  there is a comma that is throwing and error too:

return id;
},

**Comment by bajtos** [comment]
> I encountered the following jslint error:
> I believe this can be corrected by:
> ```js
> if (id == null) {
>     id = '__anonymous__';
> }

The form used in our code is a perfectly valid javascript. You need to fix the configuration of your linter to allow non-block statements after `if`. I did a quick search, it looks like jslint does not support that: http://stackoverflow.com/a/14474794/69868. 

In general, it is a bad idea to run linter on a generated code. You should exclude the generated files from linting.

> Additionally, there is a comma that is throwing and error too:
> return id;
> },

This should have been fixed by strongloop/loopback-sdk-angular#92, please upgrade your cli to you the latest version.

I am closing this issue as invalid.
